openlr/openlr_match_quality/openlr_assessment_tool/mainwindow.cpp

```diff
@@ -19,17 +26,21 @@
 
 #include <cerrno>
 #include <cstring>
+#include <vector>
 
 namespace
 {
-// TODO(mgsergio): Consider getting rid of this class: just put everything
-// in TrafficMode.
 class TrafficDrawerDelegate : public TrafficDrawerDelegateBase
 {
+  static auto constexpr kEncodedLineId = "encodedPath";
```

Comment:
Wow, wow! Please, at least for raw c-strings, don't use auto. Resutls of sizeof() might be surprising.